os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_central/ble_tizenrt_central_client_app.h

```diff
@@ -24,7 +24,7 @@ extern "C" {
  *============================================================================*/
 #include <profile_client.h>
 #include <app_msg.h>
-#include "da_ble_client.h"
+#include <tinyara/net/if/ble.h>
```

Comment:
Please add `#include <tinyara/config.h>` also.

Reply:
@jeongchanKim 
That is not the point of this PR.
This is just for removing 'da_' prefix. I will apply your opinion in next PR.